src/js/utils/index.js

```diff
@@ -21,11 +21,17 @@ export const ERROR_UNKNOWN = "unknown"
 
 export const noop = () => null
 
-export const findProjectBy = prop => val =>
-  compose(
+export const findProjectBy = prop => val => projects => {
+  if (!val) {
```

Comment:
If val is not set, return. Otherwise the first project where prop is not set will be found.